src/compiler/checker.ts

```diff
@@ -23029,6 +23029,18 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             type;
     }
 
+    function getValueOfResult(type: Type): Type {
+        const valueOf = getPropertyOfType(type, "valueOf" as __String);
+        if (valueOf) {
+            const signatures = getSignaturesOfType(getTypeOfSymbol(valueOf), ts.SignatureKind.Call);

```

Comment:
FYI annoying auto-import thing here, but should should just be `SignatureKind.Call`; sometimes the `ts` prefix appears first in the completion list and gets used (and that's one reason why I'm trying to get us to remove them everywhere).

Reply:
Main no longer has this `ts` import so when you merge from main next, you'll have to fix this (and then never have to worry about again).